client/src/stories/Component/Inputs/DatePickerInput.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import DateFnsUtils from "@date-io/date-fns";
+import {
+  MuiPickersUtilsProvider,
+  KeyboardDatePicker,
+} from "@material-ui/pickers";
+import { makeStyles } from "@material-ui/core";
+import PropTypes from "prop-types";
+
+function DatePickerInput({ idName, label }) {
+  const [selectedDate, setSelectedDate] = React.useState();
+
+  const handleDateChange = (date) => {
+    setSelectedDate(date);
+  };
+
+  const classes = useStyles();
+
+  return (
+    <div className={classes.root}>
+      <MuiPickersUtilsProvider utils={DateFnsUtils}>
```

Comment:
Providers like this could be moved to your `App.js`, because you only need one in your application. If this component is rendered multiple times, it would create multiple providers.